package.json

```diff
@@ -5,32 +5,46 @@
   },
   "description": "GitHub Desktop build dependencies",
   "scripts": {
-    "cli": "ts-node --require ./app/test/globals.ts --require ./app/src/cli/dev-commands-global.js app/src/cli/main.ts",
-    "test:integration": "cross-env TEST_ENV=1 ELECTRON_NO_ATTACH_CONSOLE=1 xvfb-maybe mocha -t 30000 --require ts-node/register app/test/integration/*.ts",
-    "test:unit": "cross-env GIT_AUTHOR_NAME=\"Joe Bloggs\" GIT_AUTHOR_EMAIL=\"joe.bloggs@somewhere.com\" GIT_COMMITTER_NAME=\"Joe Bloggs\" GIT_COMMITTER_EMAIL=\"joe.bloggs@somewhere.com\" TEST_ENV=1 ELECTRON_NO_ATTACH_CONSOLE=1 xvfb-maybe electron-mocha -t 10000 --renderer --require ts-node/register --require ./app/test/globals.ts app/test/unit/*.{ts,tsx} app/test/unit/**/*.{ts,tsx}",
-    "test": "yarn test:unit && yarn test:integration",
+    "cli":
```

Comment:
Can you also undo the formatting changes here (I guess this is VSCode's doing) so it's easier to see what's actually changed in this file?

Reply:
These changes are intentionally part of this PR — I’ve enabled Prettier for the JSON files.

Reply:
The diff for the commit (https://github.com/desktop/desktop/pull/3569/commits/88002e02b854df1cd0684c5228ca66349313cc9a) shows the semantic changes in that PR fairly well.

Reply:
> I’ve enabled Prettier for the JSON files.

Could we not do this right now? We only use JSON files in a few mission-critical places, and you're currently introducing a lot of churn in here on top of it which is making this a challenge to review.